app.js

```diff
@@ -1,20 +1,20 @@
 const express = require('express')
 const app = express()
-const { getTopics } = require('./controllers/news-controller')
-const { customErrorHandling, errorHandling500 } = require('./error-handling')
+const { getTopics, getArticles } = require('./controllers/news-controller')
+const { errorHandling400, errorHandling500, nonExistentPaths } = require('./error-handling')
 
+app.use(express.json())
 
 app.get('/api/topics', getTopics)
+app.get('/api/articles', getArticles)
 
+app.use((req, res, next) => {
```

Comment:
This is the right logic on line 12, but should be app.all('*', ()=>{}) instead. Please change